README.md

```diff
@@ -51,24 +73,77 @@ Okay, lets get down to business.
 9. You may be asked to turn off and plug in your device multiple times, the script _should_ take care of the rest.
 
 ### Linux
+
 1. Either clone this repository or download it as ZIP
```

Comment:
suggestion:
Making them all `1. xxx` can avoid unwanted git diff.
Github will convert all 1s to the correct numbers.

```md
1. aaa
1. bbb
```
gives
1. aaa
1. bbb

Reply:
Thanks for the feedback. I've seen that style before but I find putting actual numbers is good for when someone inevitably clones and runs `cat README.md`